README.md

```diff
@@ -2,27 +2,42 @@
 
 Pushing a new design and frontend concept to ownCloud
 
-![bildschirmfoto von 2018-03-14 17-28-06](https://user-images.githubusercontent.com/1005065/37416039-20817b4c-27ad-11e8-9f14-cbe12936fd64.png)
-![bildschirmfoto von 2018-03-14 17-27-40](https://user-images.githubusercontent.com/1005065/37416040-20ad906a-27ad-11e8-8a56-ad5f824743a0.png)
+![image](https://user-images.githubusercontent.com/25989331/63966638-fd4e0080-ca9b-11e9-931a-8dd9bf3ba82f.png)
+
 [See this online Demo](https://phoenix.owncloud.com/custom/phoenix/index.html#/login)
 
 ## How to build
 
 - Run `yarn install` to build core
+- Run `yarn install-all` to install dependencies of all apps and core
 - Run `yarn dist` to build all apps configured in config.json
 
+## ownCloud server prerequisites
+### When running with oauth2
+- installed and enabled oauth2 app
+- entry for Phoenix in User Authentication in admin settings with Redirection URI: `<phoenix-env>/oidc-callback.html`
+- enabled CORS domain entry for Phoenix in config.php:
+```
+'cors.allowed-domains' => [
+'<phoenix-env>',
+]
+```
+
+## Create config.json
+- For reference look into config.json.sample (example with oauth2)
+
 ## Run
 
 - optionally provide custom domain name: `export SERVER_HOST=0.0.0.0:8300`
-- run a webpack dev server `yarn run watch`
+- run a webpack dev server `yarn run watch` (`yarn run watch-all` if you want to watch apps as well)
```

Comment:
As we are omitting the `run` command in Line 12 we might as well be consistent and drop it here as well ;-)

Reply:
Makes sense. Will do